docs/setup.md

```diff
@@ -77,16 +77,15 @@ Next, several configuration files and scripts need to be tweaked:
     you wish to use an in-memory map)
         - (Optional: MDBM) `bard__mdbm_location = dir/to/mdbm` - Note that Fili assumes this directory contains a
         `dimensionCache` folder.
-    - `bard__non_ui_broker = http://url/to/druid/broker`
-    - `bard__ui_broker = http://url/to/druid/broker`
+    - `bard__druid_broker = http://url/to/druid/broker`
     - `bard__druid_coord = http://url/to/druid/coordinator`
+    - `bard__fili_port = [a free port below 65535]`
     
 * [pom.xml][pomXml] -  Find the `fili.version` tag, and update that to point to the desired version of Fili, rather
    than a snapshot. 
 
-Note that both `bard__non_ui_broker` and `bard__ui_broker` are set to the same broker URL. These parameters are 
-artifacts of the project Fili was spun out of. Eventually, these two settings will be generalized into something useful
-for other projects. For now, you can safely treat them as if they were the same.
+If you used an older generation of fili, you may have seen `bard__non_ui_broker` and `bard__ui_broker`.
+Now both of them are replaced by a unified `bard__druid_broker`. 
```

Comment:
Can probably skip this section since it's in [the changelog](https://github.com/yahoo/fili/blob/master/CHANGELOG.md#consolidate-ui--non-ui-broker-configurations)

Reply:
Sure.